src/ray/common/ray_config_def.h

```diff
@@ -488,6 +488,9 @@ RAY_CONFIG(int64_t, task_events_max_num_profile_events_for_task, 1000)
 /// GCS.
 RAY_CONFIG(uint64_t, gcs_mark_task_failed_on_job_done_delay_ms, /*  15 secs */ 1000 * 15)
 
+/// The delay in ms that GCS should mark any running tasks from a dead worker failed.
```

Comment:
Can you write a comment why we do this here?

Reply:
we could do it eagerly in my opinion actually.